less/components/btn.less

```diff
@@ -1,4 +1,8 @@
 .btn {
+  // button text should be wrapped.
+  // we shouldn't assume button text can always fit into one line,
+  // especially knowing that the site will be localized in the future
```

Comment:
> so odds are this comment will still exist far into the future when our site has been localized :)

I bet it's most likely gonna happen heh.  for that sake let me update my PR again, thanks Atul!